apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -63,11 +69,27 @@ export class RushCommandLineParser extends CommandLineParser {
         + ' automation tools.  If you are looking for a proven turnkey solution for monorepo management,'
         + ' Rush is for you.'
     });
-    const optionsIn: IRushCommandLineParserOptions = options || {};
-    this._rushOptions = {
-      cwd: optionsIn.cwd || process.cwd()
-    };
+
+    this._rushOptions = this._normalizeOptions(options || {});
+
+    try {
+      const rushJsonFilename: string | undefined = RushConfiguration.tryFindRushJsonLocation({
+        startingFolder: this._rushOptions.cwd,
+        showVerbose: true
+      });
+      if (rushJsonFilename) {
+        this.rushConfiguration = RushConfiguration.loadFromConfigurationFile(rushJsonFilename);
+      }
+    } catch (error) {
+      this._reportErrorAndSetExitCode(error);
+    }
+
     this._populateActions();
+
+    NodeJsCompatibility.warnAboutCompatibilityIssues(
```

Comment:
Should this precede `_populateActions()`?

Reply:
In most cases it shouldn't matter, but I'll move it for the edge cases.